scripts/ci/check_compliance.py

```diff
@@ -36,6 +37,28 @@
 import list_boards
 import list_hardware
 
+sys.path.insert(0, os.path.join(str(Path(__file__).resolve().parents[2]),
+                                'scripts/dts/python-devicetree/src'))
```

Comment:
I believe `os.path.join()` is not needed when using `Path` objects

```suggestion
sys.path.insert(0, str(Path(__file__).resolve().parents[2] / "scripts" / "dts" / "python-devicetree" / "src")
```

Reply:
Done.

Reply:
Has this been fixed?

Reply:
Hmm... wait, why didn't my changes appear? I changed it again.